pilot/pkg/model/config.go

```diff
@@ -36,6 +36,12 @@ const (
 	DefaultAuthenticationPolicyName = "default"
 )
 
+var (
+	// defaultAuthenticationServiceMeshPolicyNamespace is the namespace containing the mesh-scoped authentication policy.
+	// only policy in this namespace will be considered.
+	defaultAuthenticationServiceMeshPolicyNamespace = "istio-system"
```

Comment:
Remove "default" from the variable name?

Reply:
Will do

Reply:
Have updated both to remove default